site/docs/contribute/openapi.md

```diff
@@ -0,0 +1,31 @@
+# Using OpenAPI Generator to Generate Code for a Data Catalog Connector (in the Go Language)
+
+The Fybrik repository contains specification files that detail the data catalog connector API. These files include [datacatalog.spec.yaml](https://github.com/fybrik/fybrik/blob/master/connectors/api/datacatalog.spec.yaml) and [taxonomy.json](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/). They detail all the fields that the data catalog connector should expect for each of the supported operations: createAsset, getAssetInfo, deleteAsset, and updateAsset.
+
+The OpenAPI generator is a tool that can be used to generate the skeleton code for REST servers in a variety of programming languages, given a specification file (written in adherence to the [OpenAPI standard](https://swagger.io/specification/)). In our case, we used the OpenAPI generator to generate skeleton code for a Fybrik data catalog connector server, in the [go](https://go.dev/) programming language. As expected, this skeleton code does not provide any functionality, since the specification file details only the API, not the functionality. Also, the behavior of the actual connector code must surely depend on the data catalog chosen to organize the Fyrbik assets.
+
+Here is a snippet of the Makefile used to automatically generate code:
+
+```
+generate-code:
+        git clone https://github.com/fybrik/fybrik/
+        cd fybrik && git checkout v0.7.0
```

Comment:
the branch/version should be  env variable
Some explanation about env variables is required.

Reply:
Replaced v0.7.0 with environment variable.
I think that explaining all the environment variables here would be too long and confusing